content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -995,6 +995,21 @@ last saw a request where the client authenticated using the service account toke
 If a legacy token was last used before the cluster gained the feature (added in Kubernetes v1.26),
 then the label isn't set.
 
+### kubernetes.io/legacy-token-invalid-since
+
+Type: Label
+
+Example: `kubernetes.io/legacy-token-invalid-since: 2023-10-27`
+
+Used on: Secret
+
+The control plane only adds this label to auto-generated Secrets that have the type
+`kubernetes.io/service-account-token`.
+The value of this label records the date (ISO 8601 format, UTC time zone) when the control plane
+detects that the auto-generated Secret is not used in a specified time (default to be one year).
+
+The label starts to be used in Kubernetes v1.29.
```

Comment:
```suggestion
The label is introduced in Kubernetes v1.29.
```

Reply:
Even better:
```diff
-The label is introduced in Kubernetes v1.29.
+This label was introduced in Kubernetes v1.29.
```
The v1.29 docs won't be published until (shortly) after the v1.29 release has happened.

Reply:
Revised.